project/frontend/src/pages/SwipeMatchPage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+
+function SwipeMatchPage(props) {
+  const username = props.location.state.username;
```

Comment:
where is the username coming from? the profile page or is this going to be a user input?